SU-SWS/Stanford-Drupal-Profile
# Update services and SMTP

# READY FOR REVIEW

# Summary
- Security updates for `services` and `smtp`

# Needed By (Date)
- July 11th

# Criticality
- How critical is this PR on a 1-10 scale? 7

# Steps to Test

1. Look at what Travis says
2. Check out this branch
3. `find stanford-jumpstart-deployer -name *.make | xargs grep -E "services\]\[version"`
4. `find stanford-jumpstart-deployer -name *.make | xargs grep -E "smtp\]\[version"`
5. Steps 3 and 4 will ensure that it's been caught everywhere

We've already tested the services functionality, so this is just to get it into the profile.

# Affected Projects or Products
- Anchorage

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-97
- https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/150
